terraform/shared/shared.tf.json.template.py

```diff
@@ -282,6 +282,36 @@ def metric_name(self) -> str:
                 }
             }
         },
+        **(
+            {
+                'aws_cloudformation_stack': {
+                    'chatbot': {
+                        'name': config.qualified_resource_name('chatbot'),
+                        'template_body': json.dumps({
+                            'AWSTemplateFormatVersion': '2010-09-09',
+                            'Description': 'AWS Chatbot Configuration',
+                            'Resources': {
+                                'SlackChannelConfiguration': {
+                                    'Type': 'AWS::Chatbot::SlackChannelConfiguration',
+                                    'Properties': {
+                                        'ConfigurationName': config.qualified_resource_name('chatbot'),
+                                        'GuardrailPolicies': 'arn:aws:iam::aws:policy/CloudWatchReadOnlyAccess',
+                                        'IamRoleArn': '${aws_iam_role.chatbot.arn}',
+                                        'LoggingLevel': 'INFO',
+                                        'SlackChannelId': config.slack_integration['channel_id'],
+                                        'SlackWorkspaceId': config.slack_integration['workspace_id'],
```

Comment:
Another reason to prefer `attrs` over `TypedDict` is improved auto-complete for attributes are opposed to keys
```suggestion
                                        'SlackChannelId': config.slack_integration.channel_id,
                                        'SlackWorkspaceId': config.slack_integration.workspace_id,
```